internal/db/schema/migrations/oss/postgres/66/01_plugin_host_external_name.up.sql

```diff
@@ -0,0 +1,50 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: MPL-2.0
+
+begin;
+  alter table host_plugin_host
+    add column external_name text
+      -- As it currently stands, AWS restricts EC2 Instance names to 256 UTF-8
+      -- characters. Azure restricts their VM resource names to, at most, 64
+      -- characters, and also have a list of disallowed runes.
+      --
+      -- These constraints follow the lowest common denominator between both
+      -- providers for maximum compatibility. They are also synced with their
+      -- logical counterpart (see `NewHost` in internal/host/plugin/host.go) to
+      -- prevent a situation where a bad name could make an update job stop
+      -- working.
+      constraint external_name_only_has_printable_characters
+        check (external_name is null or external_name !~ '[^[:print:]]')
```

Comment:
```suggestion
        check (external_name !~ '[^[:print:]]')
```
"It should be noted that a check constraint is satisfied if the check expression evaluates to true or the null value." -- [PostgreSQL Docs](https://www.postgresql.org/docs/11/ddl-constraints.html#DDL-CONSTRAINTS-CHECK-CONSTRAINTS)

Reply:
Removed null checks with https://github.com/hashicorp/boundary/pull/3074/commits/1d1805afc46c31a23a34806fd09aa0f787d1d143